### PR TITLE
Riscv support

### PR DESCRIPTION
**Pull request by sxa**
This has got past the initial detection phase in https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3263/console although may well fall over later. See also https://github.com/AdoptOpenJDK/openjdk-systemtest/pull/349 in the ST repo

### CODE REVIEW

**Comment by sxa** [comment]
Taking back out of draft as https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/3264/console seems to be running through quite well.

File: buildenv/jenkins/openjdk_tests [hunk 1]
```diff
@@ -104,6 +104,10 @@ def PLATFORM_MAP = [
     'x86-64_windows_xl' : [
         'SPEC' : 'win_x86-64',
         'LABEL' : 'ci.role.test&&hw.arch.x86&&sw.os.windows',
+    ],
+    'riscv64_linux' : [
+        'SPEC' : 'linux_riscv-64_cmprssptrs',
```
Comment (smlambert) [inline buildenv/jenkins/openjdk_tests:109]: This should match the spec listed in openj9/buildspecs, so https://github.com/eclipse/openj9/blob/master/buildspecs/linux_riscv64_cmprssptrs.spec (no '-' in the name).

**Comment by llxia** [comment]
Just note that `SPEC` value needs to be added/matched in [ottawa.csv](https://github.com/AdoptOpenJDK/TKG/blob/master/resources/ottawa.csv) as well. Otherwise, tests with variations (i.e., ModeXXX) will not work properly.

FYI @ChengJin01

**Comment by ChengJin01** [comment]
@llxia & @ShelleyLambert ,  I already had draft changes (not yet submitted for review) at https://github.com/ChengJin01/TKG/tree/add_riscv_support_tkg, which including the change of ottawa.csv.  I will change it back to `linux_riscv64_cmprssptrs` to match the buildspec in OpenJ9.

**Comment by llxia** [comment]
Riscv change https://github.com/AdoptOpenJDK/openjdk-tests/pull/1832 is merged. Close this PR.

**Pull request closed without merging**